src/components/DataView/example/DataViewKitchenSink.tsx

```diff
@@ -672,29 +672,29 @@ function DataViewKitchenSink(): ReactElement {
 				filter
 			});
 		},
-		savedViewAllowSharedViewSave
+		savedViewAllowSharedViewSave,

```

Comment:
I'm not sure why you decided to move these props here, but they don't affect so I'll leave them here